### PR TITLE
bz2078943 updated workload partitioning procedure

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.9+

Issue:
[BZ2078943](https://bugzilla.redhat.com/show_bug.cgi?id=2078943)

Link to docs preview (VPN required):
[Enabling workload partitioning](http://file.rdu.redhat.com/antaylor/06-06-22/bz2078943/scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.html)

Additional information:
none

### CODE REVIEW

**Comment by lack** [comment]
This looks great to me!

**Comment by sheriff-rh** [comment]
@sunilcio PTAL, thanks.

File: modules/sno-du-enabling-workload-partitioning.adoc [hunk 1]
```diff
@@ -48,3 +48,33 @@ spec:
         user:
           name: root
 ----
+
+* The contents of `/etc/crio/crio.conf.d/01-workload-partitioning` should look like this.
```
Comment (tmalove) [inline modules/sno-du-enabling-workload-partitioning.adoc:52]: Replace the period with a colon, per the ISG here: https://docs.engineering.redhat.com/display/CCSENABLE/Colons+and+semicolons

**Comment by sheriff-rh** [comment]
Feedback applied, ready for merge.

**Pull request merged**